lms/templates/ux/reference/teams-base.html

```diff
@@ -17,8 +17,7 @@
     <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
       <a href="/" title="Home page" itemprop="url">
```

Comment:
OK. Please ping me when you want me to take a second look.

On Mon, Nov 2, 2015 at 1:46 PM, Mark Sadecki <notifications@github.com>
wrote:

> In lms/templates/ux/reference/teams-base.html
> <https://github.com/edx/edx-platform/pull/10347#discussion_r43664338>:
>
> > @@ -17,8 +17,7 @@
> >      <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
> >        <a href="/" title="Home page" itemprop="url">
>
> @sarina <https://github.com/sarina> I'm going to get rid of that
> attribute all together as it shouldn't even be there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10347/files#r43664338>.
>